src/parser/io/ptb_reader.cpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file ptb_reader.cpp
+ * @author Chase Geigle
```

Comment:
This looks very similar to the `analyzers::parse_tree` string constructor. It looks like the only difference is that the PTB format allows newlines between characters. If that is true, we might be able to completely remove it after adding some very basic functions to the other `parse_tree` object. Separating the reader into this file is nicer anyway.